experimental/packages/opentelemetry-exporter-prometheus/test/util.ts

```diff
@@ -15,17 +15,13 @@
  */
 
 import * as sinon from 'sinon';
-import * as perf_hooks from 'perf_hooks';
 import { Resource } from '@opentelemetry/resources';
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
 
 export const mockedHrTimeMs = 1586347902211;
 
 export function mockHrTime() {
-  // We cannot stub core.now or core.timeOrigin since a property of
-  // ModuleNamespace can not be reconfigured.ß
-  sinon.stub(perf_hooks.performance, 'timeOrigin').value(0);
-  sinon.stub(perf_hooks.performance, 'now').returns(mockedHrTimeMs);
+  sinon.useFakeTimers(mockedHrTimeMs);
```

Comment:
i could not get this to behave locally; is there a standard pattern here??

Reply:
Hmm, interesting - the fix you applied to the test works fine on my machine and in CI. Looks exactly like it's supposed to be. :thinking: Could you paste your test output?

Reply:
It could be helpful to do a full clean and rebuild. Sometimes dependencies get in a weird state

Note: this takes a while. ensure you do not have any work that is not checked in because it could be lost.

From the root of the project:

```sh
git reset --hard && lerna clean && git clean -ffdx && npm install && npm run compile
```

Reply:
oh great, this seems to do the trick! the local issue was that the prometheus exporter tests were still using the performance timer when recording metrics; but the clean + rebuild seemed to force it to update its deps and use `Date.now()` as expected. thanks!